src/vault-api/core/Reveal.ts

```diff
@@ -66,13 +66,13 @@ const getSkyflowIdRecordsFromVault = (
   });
 
   if (options && Object.prototype.hasOwnProperty.call(options, 'encodeURI') && options?.encodeURI === false){
-    skyflowIdRecord.columnValues?.forEach((column) => {
-      paramList += `column_name=${skyflowIdRecord.columnName}&column_values=${column}&`;
+    skyflowIdRecord.columnValues?.forEach((column,index) => {
+      paramList += `${index === 0 ?`column_name=${skyflowIdRecord.columnName}`:''}&column_values=${column}&`;
```

Comment:
How about taking `&` side inside ? `${index === 0 ?`column_name=${skyflowIdRecord.columnName}&`:''}` otherwise we will have lots of &&& in query parameters

Reply:
Make sense updated it